docker/maistra-builder_3.0.Dockerfile

```diff
@@ -253,9 +122,14 @@ RUN chmod 777 /go && \
     chmod 777 /home/.helm && \
     chmod 777 /home/.gsutil
 
+RUN mkdir -p /work && chmod 777 /work
+WORKDIR /work
+
+ENV PATH=/usr/lib/llvm/bin:/usr/local/google-cloud-sdk/bin:$PATH
```

Comment:
Do we really need `GCLOUD_VERSION` and google-cloud-sdk envs in this builder image ?

Reply:
yes, it's going to be used in post-submit to upload the binary to GCS.